### PR TITLE
Don't create multiple remainder outputs

### LINKED ISSUE #451: Multiple outputs for multiple custom inputs shouldn't create multiple outputs for the same address
MessageError(DuplicateError) happens when we have more inputs than we need from the same address and therefore create multiple remainder outputs

### PR DESCRIPTION
**Pull request by Thoralf-M**
Fixes #451

### CODE REVIEW

File: bindings/nodejs/lib/index.d.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export declare class Client {
   getAddressBalance(address: string): Promise<AddressBalance>
   isAddressValid(address: string): boolean
   getMilestone(index: number): Promise<MilestoneMetadata>
-  getMilestoneUTXOChanges(index: number): Promise<MilestoneUTXOChanges>
+  GetMilestoneUtxoChanges(index: number): Promise<MilestoneUTXOChanges>
```
Comment (bingyanglin) [inline bindings/nodejs/lib/index.d.ts]: A popular convention in Javascript is to only capitalize constructors, for methods the first world should be lowercase, i.e., `getMilestoneUtxoChanges`

File: bindings/nodejs/native/src/classes/client/mod.rs [hunk 2]
```diff
@@ -470,7 +470,7 @@ declare_types! {
             Ok(cx.undefined().upcast())
         }
 
-        method getMilestoneUTXOChanges(mut cx) {
+        method GetMilestoneUtxoChanges(mut cx) {
```
Comment (bingyanglin) [inline bindings/nodejs/native/src/classes/client/mod.rs]: Same as the above.

File: bindings/nodejs/lib/index.d.ts (hunk 1 above)
Comment (Thoralf-M) [reply bindings/nodejs/lib/index.d.ts]: Yeah, but Clippy complained about it, should I add some ignore rule?

Comment (bingyanglin) [reply bindings/nodejs/lib/index.d.ts]: The Clippy complained about `UTXO`, which should be `Utxo`. the `get` prefix should be OK.

Comment (Thoralf-M) [reply bindings/nodejs/lib/index.d.ts]: Oh right, sry read it wrong

Comment (bingyanglin) [reply bindings/nodejs/lib/index.d.ts]: No no don't be. This PR is quite good.

**Review by bingyanglin — APPROVED** [review]

**Pull request merged**